lib/constable/services/announcement_creator.ex

```diff
@@ -49,27 +61,27 @@ defmodule Constable.Services.AnnouncementCreator do
   defp email_users(announcement, users) do
     users = filter_author(announcement.user_id, users)
 
-    Emails.new_announcement(announcement, users) |> Mailer.deliver_later
+    Emails.new_announcement(announcement, users) |> Mailer.deliver_later()
     announcement
   end
 
   def email_mentioned_users(announcement, users) do
     users = filter_author(announcement.user_id, users)
 
-    Emails.new_announcement_mention(announcement, users) |> Mailer.deliver_later
+    Emails.new_announcement_mention(announcement, users) |> Mailer.deliver_later()
     announcement
   end
 
-  defp filter_author(author_id, users)  do
-    Enum.reject(users, fn(user) ->
+  defp filter_author(author_id, users) do
+    Enum.reject(users, fn user ->
```

Comment:
Im curious, did the formatter change this line? (removing parens) Im just trying to learn about its preferences regarding paren use :)

Reply:
Yes, there are several formatter changes in this PR. I'm not familiar with our formatter configuration, so I have just been letting it do its thing. It's something I'd love to discuss with @germsvel when he is back.